celsos_bookSearch/celsos_bookSearch/View/SearchViewController.swift

```diff
@@ -9,24 +9,64 @@ import UIKit
 
 class SearchViewController: UIViewController {
 
+    // MARK: - Outlets
+    @IBOutlet weak var searchBarOutlet: UISearchBar!
+    @IBOutlet weak var recentSearchesTableViewOutlet: UITableView!
+
+    var searchesArray: [String] = []
+    let searchCellNibName = "SearchesTableViewCell"
+    let searchCellIdentifier = SearchesTableViewCell().searchCellIdentifier
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        self.view.backgroundColor = .green
-        self.title = "Teste Nav"
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+
+        let tap = UITapGestureRecognizer(target: self, action: #selector(UIInputViewController.dismissKeyboard))
+        view.addGestureRecognizer(tap)
+
+        searchBarOutlet.delegate = self
 
-        // Do any additional setup after loading the view.
+        recentSearchesTableViewOutlet.delegate = self
+        recentSearchesTableViewOutlet.dataSource = self
+        recentSearchesTableViewOutlet.reloadData()
+
+        let searchCellNib = UINib(nibName: searchCellNibName, bundle: Bundle(for: SearchesTableViewCell.self))
+        self.recentSearchesTableViewOutlet.register(searchCellNib, forCellReuseIdentifier:                        self.searchCellIdentifier)
```

Comment:
Remove the additional space after _forCellReuseIdentifier:_

Reply:
Done